components/templated/actions/create-render/create-render.mjs

```diff
@@ -0,0 +1,38 @@
+import app from "../../templated.app.mjs";
+
+export default {
+  key: "templated-create-render",
+  name: "Create Render",
+  description: "Creates a render on a template in Templated. [See the documentation](https://app.templated.io/docs#create-render)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    app,
+    templateId: {
+      propDefinition: [
+        app,
+        "templateId",
+      ],
+    },
+    layers: {
+      label: "Layers",
+      description: "An object of layers that will be updated in the template. E.g. { \"text-1\" : { \"text\" : \"This is my text to be rendered\", \"color\" : \"#FFFFFF\", \"background\" : \"#0000FF\" }, \"image-1\": { \"image_url\" : \"https://pathtomyphoto.com/123.jpg\" } }",
+      type: "object",
+    },
+  },
+  async run({ $ }) {
+    const layers = typeof this.layers === "string"
+      ? JSON.parse(this.layers)
+      : this.layers;
+
+    const response = await this.app.createRender({
+      $,
+      id: this.templateId,
+      layers,
```

Comment:
Is this supposed to be passed as `layers`? Shouldn't it be in the request body instead? e.g. `data: { layers }`